gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/integration/use_case/CreateIntegrationUseCaseTest.java

```diff
@@ -83,6 +86,9 @@ public class CreateIntegrationUseCaseTest {
     RoleQueryServiceInMemory roleQueryServiceInMemory = new RoleQueryServiceInMemory();
     UserCrudServiceInMemory userCrudServiceInMemory = new UserCrudServiceInMemory();
     GroupQueryServiceInMemory groupQueryServiceInMemory = new GroupQueryServiceInMemory();
+    MembershipQueryServiceInMemory membershipQueryService = new MembershipQueryServiceInMemory();
+    GroupQueryServiceInMemory groupQueryService = new GroupQueryServiceInMemory();
+    UserCrudServiceInMemory userCrudService = new UserCrudServiceInMemory();
```

Comment:
If we move the instantiation outside the `beforeEach` we should reset them in the `afterEach`